drivers/wifi/infineon/airoc_wifi.c

```diff
@@ -364,6 +364,35 @@ static void airoc_wifi_network_process_ethernet_data(whd_interface_t interface,
 	}
 }
 
+static enum ethernet_hw_caps airoc_get_capabilities(const struct device *dev)
+{
+	ARG_UNUSED(dev);
+
+	return ETHERNET_HW_FILTERING;
+}
+
+static int airoc_set_config(const struct device *dev,
+				  enum ethernet_config_type type,
+				  const struct ethernet_config *config)
+{
+	ARG_UNUSED(dev);
+	whd_mac_t whd_mac_addr;
+
+	switch (type) {
+	case ETHERNET_CONFIG_TYPE_FILTER:
+		for (int i = 0; i < WHD_ETHER_ADDR_LEN; i++)
```

Comment:
Please follow zephyr coding style, so always use {} even for one liners

Reply:
updated